pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -695,6 +695,23 @@ func getServersConfig(cr *api.KieApp) ([]api.ServerTemplate, error) {
 			}
 
 			// Set replicas
+			if len(cr.Spec.Objects.Servers) == 0 {
+				serverSet.Replicas = serverReplicas
+			} else {
```

Comment:
This is already been handled below at -
https://github.com/kiegroup/kie-cloud-operator/blob/3c2e2b8e48d544e07922192c7497cc15c10c7d70/pkg/controller/kieapp/defaults/defaults.go#L697-L700

Reply:
Ok, simplified